lib/internal/loader/Translators.js

```diff
@@ -90,3 +90,15 @@ translators.set('json', async (url) => {
     }
   });
 });
+
+translators.set('wasm', async (url) => {
+  const bytes = await readFileAsync(new URL(url));
```

Comment:
The other translators have a "debug" line that outputs the action (i.e. "loading wasm ${url}"). I would suggest adding it here.